javascript/src/main/pages/Admin/Teams.js

```diff
@@ -1,46 +1,29 @@
 import React from "react";
-// import BootstrapTable from 'react-bootstrap-table-next';
+import BootstrapTable from 'react-bootstrap-table-next';
 // import { Button } from "react-bootstrap";
-import Table from 'react-bootstrap/Table'
-// import useSWR from "swr";
-// import {useAuth0} from "@auth0/auth0-react";
-// import {fetchWithToken} from "../../utils/fetch";
+import useSWR from "swr";
+import {useAuth0} from "@auth0/auth0-react";
+import {fetchWithToken} from "../../utils/fetch";
 
 const Teams = () => {
-    // const { getAccessTokenSilently: getToken } = useAuth0();
-    //const { data: slackUsers } = useSWR(["/api/slackUsers", getToken], fetchWithToken);
+    const { getAccessTokenSilently: getToken } = useAuth0();
+    const { data: teamNames } = useSWR(["/api/member/teams", getToken], fetchWithToken)
     
-/*    const { data: slackUsers };
-*/    // TODO: update with information from the CSV file, NOT from slackUsers
-    /*const columns = [{
-        dataField: 'name',
-        text: 'Name'
-    }, {
-        dataField: 'team',
-        text: 'Team'    
-    }];*/
+    const columns = [{
+        dataField: 'teamName',
+        text: 'Team'
+    },
+    {
+        dataField: 'id',
+        text: 'id'
+    },
+    {
+        dataField: 'teamDescription',
+        text: 'Description'
+    }];
 
-    
     return (
-        //<BootstrapTable keyField='id' data={slackUsers || []} columns={columns} />
-        <Table striped bordered hover>
-            <thead>
-                <tr>
-                    <th>Name</th>
-                    <th>Team</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr>
-                    <td>PlaceHolder1</td>
-                    <td>TeamPlaceHolder</td>
-                </tr>
-                <tr>
-                    <td>PlaceHolder2</td>
-                    <td>TeamPlaceHolder</td>
-                </tr>
-            </tbody>
-        </Table>
+        <BootstrapTable keyField='teamName' data={teamNames || []} columns={columns} />

```

Comment:
There seems to be no tests associated with these lines of code. The tests present just makes sure it renders, which is insufficient. I think for this to be merged, there should be better tests written, but don't worry about it too much.

Reply:
I checked the slackUsers.test.js doesn't contain any testcase either. So I just leave the test case like that.

Reply:
ok that's fine I guess.